src/Splat.NLog/NLogLogger.cs

```diff
@@ -14,6 +14,7 @@ namespace Splat.NLog;
 [DebuggerDisplay("Name={_inner.Name} Level={Level}")]
 public sealed class NLogLogger : IFullLogger, IDisposable
 {
+    private static readonly LogLevel[] _allLogLevels = Enum.GetValues(typeof(LogLevel)).Cast<LogLevel>().ToArray();
```

Comment:
Use the Enum.GetValues<T> override, it's AOT compatible for future proofing.

Reply:
`Enum.GetValues<T>` is only available for NET5 (and newer). Added new commit.

Reply:
Yeah caching is good. In AOT land they actually compile it to an array, but in non-AOT land its standard behavior.